priv/repo/migrations/20160707161837_add_revoked_to_keys.exs

```diff
@@ -0,0 +1,11 @@
+defmodule HexWeb.Repo.Migrations.AddRevokedToKeys do
+  use Ecto.Migration
+
+  def change do
+    alter table(:keys) do
+      add :revoked_name, :text
```

Comment:
We can change the unique constraint to be on the pair of `(name, revoked_at)` to workaround this issue.